README.md

```diff
@@ -23,27 +23,53 @@ You can watch Nodemon do it’s thing if you run `docker logs -f selfservice`.
 
 If you’re making changes to client-side JS or Sass files (anything within [`/browsered/`](https://github.com/alphagov/pay-selfservice/tree/BAU-update-README-to-explain-livereload/app/browsered) or [`/assets/`](https://github.com/alphagov/pay-selfservice/tree/BAU-update-README-to-explain-livereload/app/assets)) then running `npm run watch-live-reload` will watch for changes and recompile. Nodemon does not do anything here as that’s not necessary. If you install the [livereload browser plugin](http://livereload.com/extensions/) then it will refresh your page once the assets have been compiled to the `/public` folder.
 
-### Steps to run locally - new method
-The new method is faster and easier to run in debug mode.
-* `./scripts/generate-dev-environment.sh local` - this will create a local `.env` file.
-* Repeat the same steps above.
-* Run the app with: `npm run start:dev`
-* Go to: http://localhost:3000 -> Note the diff port number.
-
-#### Run in debug mode
-* You need to make sure the app runs locally first by using the instructions above.
+### Run and debug locally - new method
```

Comment:
I think I would be in favour of showing how running locally works as of right now (how these both work together) rather than splitting into new and old methods. 

I haven't had the time to think this all the way through but the breakdown might be something like: 

---



# Running locally

## Getting set up

```bash
npm install 
npm run compile
```

## Running

Self service reads from environment variables that allow it to connect to backend services from any environment, there are scripts to create an environment file to point to different backends.

### Tunnel

Tunneling into different test and staging environments allows you to test with real data. 

```bash
node scripts/generate-dev-environment tunnel
npm run start:dev
```

### Local

If you have the backend services running locally in docker, you can connect directly to them. 

```bash
# This CLI tool is only available internally at the moment 
pay local up --cluster endtoend
node scripts/generate-dev-environment local
npm run start:dev
```

---

This way around seems a bit clearer to me and it never recommends running self service in Docker which is considerably slower on most machines.

Reply:
I have added your suggestions - please take a look and let me know what you think.